code/src/utest/plugin/lsttokens/choose/SchoolsTokenTest.java

```diff
@@ -47,6 +50,7 @@
 	static CDOMTokenLoader<CDOMObject> loader =
 			new CDOMTokenLoader<>();
 
+	@BeforeEach
```

Comment:
This is inconsistent with files above.  We should choose what of override and beforeeach goes first

Reply:
I think fixing this across the board requires a separate PR. In the meantime I'll go with BeforeEach first (alphabetical?).